### PR TITLE
Fix for #26 - Removed the SpellcheckerWorker and made use of Spellche…

### PR DESCRIPTION
**Pull request by Devmacs**
…ckerWasm directly in the CLI script. Updated Readme with new NodeJS usage.

All tests passing, removed the Worker test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,57 +88,24 @@ function resultHandler(results) {
...
 ## Usage as a Node Worker
 ```typescript
-import { Worker, MessageChannel, MessagePort } from 'worker_threads';
-import { deserializeSuggestedItems } from './utils';
+const { SpellcheckerWasm } = require('../lib/nodejs/SpellcheckerWasm.js');
 
-const wasmPath = require.resolve('spellchecker-wasm/lib/spellchecker-wasm.wasm');
-const dictionaryLocation = require.resolve('spellchecker-wasm/lib/frequency_dictionary_en_82_765.txt');
+const wasmPath = resolve(__dirname, '../lib/spellchecker-wasm.wasm');
+const dictionaryPath = resolve(__dirname, '../lib/frequency_dictionary_en_82_765.txt');
```
Comment (justinwilaby) [inline README.md]: These examples are setup assuming an installed library in `node_modules`. Using `__dirname` here assumes a cloned repo. Reverting this back to using `require.resolve()` should keep it consistent.

Comment (justinwilaby) [reply README.md]: @Devmacs - ^^^ Let's get this merged in! Are you able to make the updates I suggested?

Comment (Devmacs) [reply README.md]: Sorry for the delay, it's been quite busy over here. I'll take care of this later today; thanks for the reminder.

Comment (Devmacs) [reply README.md]: Done. Please let me know if there's anything else you'd like me to change. Thanks!

**Comment by justinwilaby** [comment]
Looks good to me. Thank you!

**Pull request merged**